sdk/digitaltwins/digital-twins-core/src/digitalTwinsClient.ts

```diff
@@ -196,7 +196,7 @@ export class DigitalTwinsClient {
    */
   public updateDigitalTwin(
     digitalTwinId: string,
-    jsonPatch: any,
+    jsonPatch: any, // eslint-disable-line @typescript-eslint/explicit-module-boundary-types
```

Comment:
I added this because actually addressing the issue would change the public surface. Thoughts? @ramya-rao-a

Reply:
Since the package is still in preview, changing API surface for the better is fine.
You can consider `unknown` if you do not have a type you can make for this.

Reply:
The package is no longer in preview. In this case, should we just leave it as-is? @ramya-rao-a

Reply:
Ah! My mistake. 

I still believe changing it to `unknown` for input arguments is not a breaking change. User can still pass  "any"thing and it will work. If we were changing return type from `any` to `unknown`, that would be a breaking change as the user might be accessing properties on the return object without errors before and they will start getting the errors now.

@chradek Do you agree?

@vishnureddy17 while we discuss this, for now, can you update the comment as below for future reference?

```suggestion
    jsonPatch: any, // eslint-disable-line @typescript-eslint/explicit-module-boundary-types -- changing the type any would be a breaking change
```

Reply:
I agree that changing `any` to `unknown` is safe on the __input__ side of an API, but not the __output__ side. That said, I think this is just getting forwarded to the generated code, so I'm not sure there's much gained by making the change aside from keeping the linter happy (but that's the goal here :smile:)

Using `unknown` is better than any when you're working with the parameter because it forces you to make assertions about what the parameter is before you use it. In this case since we're only forwarding it, I don't see any additional benefit. That said it doesn't hurt either.